### ISSUE TITLE
Add support for "managed_schema" for the "browser.storage.managed" API

### ISSUE THREAD

**Issue by anatoli26** [mozilla/policy-templates#904]
Hi Michael,

Could you please have a look at this request about subj: https://bugzilla.mozilla.org/show_bug.cgi?id=1771731?

Is it possible?

Thanks

**Comment by mkaply** [comment]
Yes, thanks for letting me know

**Comment by anatoli26** [comment]
Michael, so do you see this possible to implement? Any ETA?

**Comment by anatoli26** [comment]
Also, just to let you know what other add-on devs say about implementing managed storage: https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/issues/1394#issuecomment-1138232866.

Maybe to increase adoption there could be some sort of FAQ and/or a code sample that could be used directly in a add-on that would address the common questions, like those in the link above?

**Comment by mkaply** [comment]
Since this is a Firefox feature, the discussion belongs in the bug, not here. Can you repost your comments there?

And we do have code samples

https://extensionworkshop.com/documentation/enterprise/adding-policy-support-to-your-extension/

**Comment by anatoli26** [comment]
Michael, I've just submitted a comment in the bugzilla issue tracker, I'll follow the subj there.

But just seizing this issue, an unrelated question: if I understand it correctly, you are also implementing something similar for Thunderbird at https://github.com/thundernest/policy-templates. But in that repo there are no issues available, so one can't ask questions. I was able to use policies that way also in Thunderbird, everything works as expected, but I have some questions and suggestions. Where should I submit them?
